syz-verifier/main.go

```diff
@@ -242,20 +256,83 @@ func startRPCServer(vrf *Verifier) (*RPCServer, error) {
 }
 
 // Connect notifies the RPCServer that a new Runner was started.
-func (srv *RPCServer) Connect(a *rpctype.RunnerConnectArgs, r *int) error {
+func (srv *RPCServer) Connect(a *rpctype.RunnerConnectArgs, r *rpctype.RunnerConnectRes) error {
 	srv.mu.Lock()
 	defer srv.mu.Unlock()
 	pool, vm := a.Pool, a.VM
 	srv.pools[pool].vmRunners[vm] = nil
+	r.CheckUnsupportedCalls = !srv.pools[pool].checked
+	return nil
+}
+
+// CheckUnsupported communicates to the server the list of system calls not
+// supported by the kernel corresponding to this pool and updates the list of
+// enabled system calls. This function is called once for each kernel.
+// When all kernels have reported the list of unsupported system calls, the
+// choice table will be created using only the system calls supported by all
+// kernels.
+func (srv *RPCServer) CheckUnsupported(a *rpctype.CheckUnsupportedArgs, r *int) error {
```

Comment:
I stumbled on this again, and I confused myself: I thought CheckUnsupported is called by the server to the runner, because "Check" implies some active checking. But it's the other way around. In that case, shouldn't this be UpdateUnsupported? As in "runner updates list of unsupported syscalls on server"?